packages/lit-html/src/directives/style-map.ts

```diff
@@ -64,37 +64,41 @@ class StyleMap extends Directive {
   update(part: AttributePart, [styleInfo]: DirectiveParameters<this>) {
     const {style} = part.element as HTMLElement;
 
-    if (this.previousStyleProperties === undefined) {
-      this.previousStyleProperties = new Set();
+    if (this._previousStyleProperties === undefined) {
+      this._previousStyleProperties = new Set();
       for (const name in styleInfo) {
-        this.previousStyleProperties.add(name);
+        this._previousStyleProperties.add(name);
       }
       return this.render(styleInfo);
     }
 
     // Remove old properties that no longer exist in styleInfo
     // We use forEach() instead of for-of so that re don't require down-level
     // iteration.
-    this.previousStyleProperties!.forEach((name) => {
-      if (!(name in styleInfo)) {
-        this.previousStyleProperties!.delete(name);
-        if (name.indexOf('-') === -1) {
+    this._previousStyleProperties!.forEach((name) => {
+      // If the name isn't in styleInfo or it's null/undefined
+      if (styleInfo[name] == null) {
+        this._previousStyleProperties!.delete(name);
+        if (name.includes('-')) {
```

Comment:
Are we sure we're covered for this in IE11?

Reply:
It's included in the core-js polyfills.